filebeat/tests/system/test_modules.py

```diff
@@ -114,6 +115,9 @@ def run_on_file(self, module, fileset, test_file, cfgfile):
             pass
         self.wait_until(lambda: not self.es.indices.exists(self.index_name))
 
+        os.environ['TZ'] = 'Etc/UTC'
```

Comment:
If this is only intended for the running beat we should set this in `subprocess.Popen(env=...)` to avoid other effects.

Reply:
Good point, I changed it in https://github.com/elastic/beats/pull/12120/commits/bc5c1941c19233411e5a2cce78c49d3505334f81

Reply:
Great, thanks